realm-transformer/src/main/java/io/realm/transformer/RealmAnalytics.java

```diff
@@ -68,6 +68,7 @@
             + "      \"Anonymized MAC Address\": \"%USER_ID%\",\n"
             + "      \"Anonymized Bundle ID\": \"%APP_ID%\",\n"
             + "      \"Binding\": \"java\",\n"
+            + "      \"Uses Kotlin\": \"%USES_KOTLIN%\",\n"
```

Comment:
I think we need to check with @timanglade about the exact wording for this, especially as we might want to track other JVM languages like Groovy and potentially Scala.